src/ucp/core/ucp_request.h

```diff
@@ -319,6 +319,7 @@ struct ucp_request {
                     /* Progress ID, if it's UCS_CALLBACKQ_ID_NULL, no operations
                      * are in-progress */
                     uct_worker_cb_id_t cb_id;
+                    ucp_rsc_index_t    rsc_index;
```

Comment:
👍 
pls add comment to this field (like in other fields above)

Reply:
done